collections/_article/godot-at-gamescom-2023.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "gamescom, here we go!"
+excerpt: "Join us next week at the Godot booth during gamescom in Cologne from Wednesday the 23rd, to Friday the 25th. Don't miss the chance to say hello to the Godot team!"
+categories: ["events"]
+author: Emi
+image: /storage/blog/gamescom/2023/gamescom-cover.webp
+date: 2023-08-17 15:57:35
+---
+
+Next week, we'll be attending our first [gamescom](https://www.gamescom.global) in Cologne, Germany. We'll be at the Godot booth in the Business Area, Hall 2.2, Stand C-042.
+
+gamescom is a trade fair for video games held annually at the Koelnmesse in Cologne, North Rhine-Westphalia. It is the world's largest gaming event (measured by exhibition space and number of visitors), with 370,000 visitors and 1037 exhibitors from 56 countries attending the show in 2018.
+
+Thanks to [Ramatak](https://ramatak.com/) and [W4 Games](https://w4games.com/) who are donating it, the Godot Engine will have a booth. This is an excellent opportunity to meet us, and like-minded folks. Come hang out around the booth and get some free swag!
+
+
+## Event details
+You can visit the [official website](https://www.gamescom.global/en/fair-visit) for more information about the event.
+
+### 🗺️ Location
+[Koelnmesse](https://www.koelnmesse.de/), Messepl. 1, 50679 Köln, Germany ([Show on map](https://www.openstreetmap.org/?mlat=50.94590&mlon=6.98034map%3D16%2F50.9441%2F6.9798#map=16/50.9455/6.9805&layers=N))
+
+### ⏰ Opening hours:
+- Wednesday 13:00 - 19:00
+- Thursday 10:00 - 20:00
+- Friday 10:00 - 20:00
+
+### 📍 Where is the stand located?
```

Comment:
"Booth" instead of "stand" to be consistent with rest of article. Plus adding "Godot" sounds a little better, I think.
```suggestion
### 📍 Where is the Godot booth located?
```